modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb

```diff
@@ -80,7 +80,11 @@ def upload
           @power_of_attorney.reload
 
           # If upload is successful, then the PoaUpater job is also called to update the code in BGS.
-          ClaimsApi::PoaVBMSUploadJob.perform_async(@power_of_attorney.id)
+          if Flipper.enabled?(:lighthouse_claims_api_poa_use_bd)
+            ClaimsApi::V1::PoaFormBuilderJob.perform_async(@power_of_attorney.id)
+          else
+            ClaimsApi::PoaVBMSUploadJob.perform_async(@power_of_attorney.id)
```

Comment:
After successful switch to BD, we can likely delete this job and spec.

Reply:
Agreed. We'll definitely want to start cleaning up anything we're not using. We have some remnants from former moves as well that we'll need to look to clean up. so be a good time to keep active 👀